ui/components/layout/SpaceBackground.tsx

```diff
@@ -286,57 +307,81 @@ export default function SpaceBackground() {
     )
   }
 
+  // Page Visibility API - pause animations when tab is hidden
```

Comment:
maybe we can pull this out to it's own file. seems like it's repeated a bunch of times